app/ui-react/syndesis/src/modules/connections/pages/create/ConnectorsPage.tsx

```diff
@@ -0,0 +1,69 @@
+import { WithConnectors } from '@syndesis/api';
+import {
+  ConnectionCard,
+  ConnectionCreatorLayout,
+  ConnectionsGrid,
+  ConnectionsGridCell,
+  ConnectionSkeleton,
+} from '@syndesis/ui';
+import { getConnectionIcon, WithLoader } from '@syndesis/utils';
+import * as React from 'react';
+import { PageTitle } from '../../../../shared';
+import { ConnectionCreatorBreadcrumbs } from '../../components';
+import resolvers from '../../resolvers';
+
+export default class ConnectorsPage extends React.Component {
+  public render() {
+    return (
+      <WithConnectors>
+        {({ data, hasData, error }) => (
+          <>
+            <PageTitle title={'Select connector'} />
+            <ConnectionCreatorLayout
+              header={<ConnectionCreatorBreadcrumbs step={1} />}
+              content={
+                <ConnectionsGrid>
+                  <WithLoader
+                    error={error}
+                    loading={!hasData}
+                    loaderChildren={
+                      <>
+                        {new Array(5).fill(0).map((_, index) => (
+                          <ConnectionsGridCell key={index}>
+                            <ConnectionSkeleton />
+                          </ConnectionsGridCell>
+                        ))}
+                      </>
+                    }
+                    errorChildren={<div>TODO</div>}
```

Comment:
I wonder if there's a nice way to track these TODOs, I see more and more of them.  I like 'em too, but they're easy to forget.

Reply:
> I wonder if there's a nice way to track these TODOs, I see more and more of them. I like 'em too, but they're easy to forget.

Yeah, I was thinking something like [this](https://github.com/JasonEtco/todo) but I'd rather have a single mega issue with all the todos listed as, you know, todos rather than many issues like this bot does.